### DISCUSSION TITLE
Error export table data from PostgreSQL database connection.

### DISCUSSION THREAD

**Discussion by shibasis3k** [Budibase/budibase#5982] [Help]
From my app which is connected with the PostgreSQL db in the backend, Whenever I am trying to export a data using button action, it is showing the error. 
![image](https://user-images.githubusercontent.com/101543194/169342680-f8de6beb-34be-4e20-a458-a491cb3f9bb4.png)

**Comment by melohagan** [comment]
Hi @shibasis3k 

Could you share any errors from the network and console tabs please? (Fn+F12 on Chrome)

What is the schema of your postgres table?

**Reply by shibasis3k** [reply]
Thanks for the reply!!
The error in the console panel is as follows...

<Error>
<Code>AccessDenied</Code>
<Message>Access Denied.</Message>
<BucketName>favicon.ico</BucketName>
<Resource>/favicon.ico</Resource>
<RequestId>16F0FFE2CBF945B0</RequestId>
<HostId>82e6cceb-71eb-4c3d-9893-490d0b63ee3e</HostId>
</Error>

In postgres table it is a single table only which I am able to upload and export from the database connection page but not from UI.

**Reply by shibasis3k** [reply]
![image](https://user-images.githubusercontent.com/101543194/169633119-80f13d8b-e1a9-4663-b1cd-eb6f94ac2257.png)

**Reply by melohagan** [reply]
Thanks for the info. It appears there's a Bad Request. If you click on the Network tab, could you please show me the payload and response?

E.g. 
![Screenshot 2022-05-23 at 09 18 44](https://user-images.githubusercontent.com/101575380/169775478-918b9e72-6b9e-41c1-8a2a-c355e6544ecb.png)

I'm having issues myself, so it might be a bug.

**Reply by shibasis3k** [reply]
{message: "ctx.request.body.map is not a function", status: 400}
message: "ctx.request.body.map is not a function"
status: 400

**Comment by melohagan** [comment]
Hi @shibasis3k 

Has this issue been resolved in the latest version of Budibase, or are you still having problems with it?
